release-schema.json

```
@@ -137,6 +128,28 @@
           },
           "uniqueItems": true,
           "minItems": 1
+        },
+        "nonElectronicSubmission": {
+          "title": "Non-electronic submission",
+          "description": "Information about non-electronic submission requirements.",
+          "type": "object",
+          "properties": {
+            "rationale": {
+              "title": "Non-electronic submission rationale",
```

These words are now covered by the enclosing object.

```suggestion
              "title": "Rationale",
```